zeppelin-interpreter/pom.xml

```diff
@@ -51,6 +52,21 @@
 
   <dependencies>
 
+    <!-- https://mvnrepository.com/artifact/org.eclipse.lsp4j/org.eclipse.lsp4j -->
+    <dependency>
+      <groupId>org.eclipse.lsp4j</groupId>
```

Comment:
![image](https://user-images.githubusercontent.com/3612566/43108796-9789c9c8-8f1e-11e8-894b-354377173190.png)

It has `guava` as a dependency. I'm worried that it might make a problem for other interpreters.

Reply:
We should not put lsp4j in zeppelin-interpreter. It is better to put it into python module

Reply:
I agree to move it under python. Even if LSP supports most of the languages, `zeppelin-interpreter` influences many of interpreters. We should be careful when we add code in `zeppelin-interpreter`.

Reply:
It's right of course.
But this code seems reusable in other interpreters. And it would be strange to add it in python module and use it in other interpreters like method from python module.